src/main/java/pl/piekoszek/gorskimatches/challange/Challenge.java

```diff
@@ -0,0 +1,83 @@
+package pl.piekoszek.gorskimatches.challange;
+
+import javax.persistence.CascadeType;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.OneToMany;
+import java.util.List;
+import java.util.UUID;
+
+@Entity
+public class Challenge {
+    @Id
+    private UUID uuid;
+
+    private String email;
```

Comment:
Why do you prefer `private` fields + `public` getter and setter methods, rather than `public` fields?

Reply:
I think when I set it up to private field, I had some problems with setting the value of registeredUserScore or something like that. And didn't know how to make it work, so used getters and settters methods as there are easier for me.